module/VuFind/sql/mysql.sql

```diff
@@ -260,7 +260,7 @@ CREATE TABLE `user_list` (
 CREATE TABLE `user_resource` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `user_id` int(11) NOT NULL,
-  `resource_id` int(11) NOT NULL,
+  `resource_id` int(11) DEFAULT NULL,
```

Comment:
I'm confused; why are you changing the constraint on user_resource rather than on resource_tags?

Do we really need to overload existing tables, or could we create a new user_list_tags table to capture this information in a clearer way?